### ISSUE TITLE
Multiple topics in a group not appearing as separate 

### ISSUE THREAD

**Issue by GameEntity903** [TGX-Android/Telegram-X#360]
1. Telegram X 0.25.6.1620-arm64-v8a (66e9c711)
TDLib: 1.8.12 (tdlib/td@b5fe088)
Android: 11 (30)
Device: Xiaomi Xiaomi Mi A3 (qssi-user 11 RKQ1.200903.002 V12.0.26.0.RFQMIXM)
Screen: 1475x720 (density: 2.0, fps: 60.000004)
Build: `Xiaomi/laurel_sprout/laurel_sprout:11/RKQ1.200903.002/V12.0.26.0.RFQMIXM:user/release-keys`
Package: org.thunderdog.challegram
Locale: en_IN (app: en)
Installer: Google Play
APK Fingerprint: `66:46:21:34:34:5a:6a:da:c3:c1:d5:ae:a9:ce:f0:42:1b:7c:ab:68`
3. *Expected* 
![Screenshot_20230323-082414~2](https://user-images.githubusercontent.com/65657986/227089040-825ae443-97bc-4cfb-81da-7ce00f1ddd24.png)
The two different topics should be split into two different chats like it is in Telegram Main.

    *Observed*
![Screenshot_20230323-082409~2](https://user-images.githubusercontent.com/65657986/227089100-ddd8be0a-d844-4b8b-9d4a-dcc85d669088.png)
All the messages are viewed in the same group topic.

4. Telegram X 0.25.6.1620-arm64-v8a

**Comment by GameEntity903** [comment]
As a side note, Topics for groups can not be created in Telegram X. Are Topics as a whole not supported in Telegram X?

**Comment by RememberTheAir** [comment]
Topics are not yet supported in TGX

**Comment by Anisotropic** [comment]
When topics would be supported?

**Comment by urkbio** [comment]
> When topics would be supported?

I also waiting for this

**Comment by fkarg** [comment]
Still true for version `0.25.9.1637` (f3309e4d), build from 2023-06-09

**Comment by RememberTheAir** [comment]
it's pointless to point out that topics are not supported by builds that do not explicitly mention topics support in their changelog :/

**Comment by fkarg** [comment]
The intention was to 1) draw some attention to it (seems to have worked perfectly ;)) 2) provide information for people looking into this later that nothing had changed in the time between the opening of the issue and my comment 3) be this a subtle variant of 'I'd appreciate it if this were to be fixed' and 4) to receive notifications on progress.

**Comment by sussum2432** [comment]
Still not working

**Comment by Mechite** [comment]
See #345 - They're going to close your issue at some point.

**Comment by Ramshu** [comment]
+1
It's an essential feature, I hope you support it soon.

**Comment by Mechite** [comment]
Please react with the thumbs up instead of posting +1s

**Comment by SimoLRepo** [comment]
> Please react with the thumbs up instead of posting +1s

@Mechite Both. There is actually a reason for this.
Adding the "+1" has two advantages:
- The post is shown among recently edited discussions.
- More users are shown, so that the level of interest in that feature is understood

**Comment by RememberTheAir** [comment]
> The post is shown among recently edited discussions.

and why would that be desired?

> More users are shown, so that the level of interest in that feature is understood

The developer is perfectly aware that this is an highly demanded feature and that TGX will be unusable for many people until topics are fully supported. Adding noise to this issue is not going to accelerate the development, and one might also argue that pushing it with recurring useless pings might have the opposite effect

**Comment by Mechite** [comment]
Not a huge problem or something
I really think giving designs for how this would be implemented is probably what they desire
Flooding the thread makes it harder for anyone who does such a contribution to be seen

**Comment by SimoLRepo** [comment]
@RememberTheAir I agree with you but liking the [comment](#issuecomment-1962960288) of @Mechite has no benefit... Maybe it would be desirable to add likes in the [first comment](#issue-1636770686).

**Comment by MartinX3** [comment]
A year later and no topics.
I switched to the open source client "Forkgram" installed via F-Droid.
Topics are working and the app is fast.

**Comment by SimoLRepo** [comment]
> A year later and no topics. I switched to the open source client "Forkgram" installed via F-Droid. Topics are working and the app is fast.

You are right, unfortunately, however, Forkgram has the same lock-locked invasive premium features as the classic version. The Telegram X version while having the premium features does not annoy the user. For the time being, I use Telegram FOSS☹

**Comment by MartinX3** [comment]
> > A year later and no topics. I switched to the open source client "Forkgram" installed via F-Droid. Topics are working and the app is fast.
> 
> You are right, unfortunately Forkgram has the same lock-locked invasive premium features via padlock icons as the classic version. The Telegram X version while having the premium features does not annoy the user. For the time being, I use Telegram FOSS ☹. I still trust the topic implementation.

That's true, because it's a fork of the Telegram App.
But it's due to the maintainer of this fork as no time to hide it.
See https://github.com/forkgram/TelegramAndroid/issues/230

But on the other side he's fast with rebasing Forkgram on the newest Telegram App Version and also he keeps his commit history clean as a rebase on the latest Forkgram commits.

**Comment by SimoLRepo** [comment]
> That's true, because it's a fork of the Telegram App.
But it's due to the maintainer of this fork as no time to hide the premium stuff.

Exactly, maybe I'll go back to using Forkgram too. Telegram FOSS seems a little behind on the versions. It's a real shame that labels remain enabled in saved messages, as indicated in [the issue opened by me that you mentioned](https://github.com/forkgram/TelegramAndroid/issues/230). Thank you.

**Comment by MartinX3** [comment]
Woopsi, I didn't see that it was your issue ticket. :D

It seems that the FOSS and Fork guys should merge their forks and work together.
Both don't have much time.

**Comment by t0ma5** [comment]
The lack of _Multiple topics in groups_ is a real deal-breaker :(

**Comment by dsg0tm** [comment]
Same here. I need to revert back to regular telegram app due to this.

**Comment by aliuq** [comment]
Looking forward to this feature very much ☺️

**Comment by maximlunegov** [comment]
Still waiting for multiple topics.

**Comment by t0ma5** [comment]
Still waiting...

**Comment by sussum2432** [comment]
This app is almost useless at this point , still no topics?

**Comment by m-fedosov** [comment]
There is still no topics in telegram X 🤯 please, add that feature

**Comment by klueman** [comment]
This is an essential feature now. If this isn't being worked on, this app may as well be abandoned.

**Comment by m0xsec** [comment]
ditto to the call to fix this; it's being used widely across the Telegram ecosystem now

**Comment by maximlunegov** [comment]
I must have original TG just for two features video calls and this one, topics in groups.

**Comment by smiba** [comment]
Honestly close to switching to a different client, topics are so common now that it's seriously hurting some of the groups I'm in by not being able to use them

**Comment by andersrh** [comment]
This is an amazing app. Just found out that this feature is not supported. So sad because I really use topics a lot.
